sql/sql_table.cc

```diff
@@ -5443,12 +5493,11 @@ bool mysql_create_table(THD *thd, TABLE_LIST *create_table,
     promote_first_timestamp_column(&alter_info->create_list);
 
 #ifdef WITH_WSREP
-  if (thd->lex->sql_command == SQLCOM_CREATE_SEQUENCE &&
-      WSREP(thd) && wsrep_thd_is_local_toi(thd))
-  {
-    if (wsrep_check_sequence(thd, create_info->seq_create_info))
+  // CREATE SEQUENCE ... Galera cluster supports only InnoDB-sequences
+  if (WSREP(thd) && wsrep_thd_is_local_toi(thd) &&
+      thd->lex->sql_command == SQLCOM_CREATE_SEQUENCE &&
```

Comment:
I suppose TOI replication has already started here, and  wsrep_thd_is_local_toi() affects only local TOI execution. How does the applier filter out unsupported sequence engine use?

Reply:
It is done in wsrep_check_support before TOI is started (there is no condition like sql_command == something) instead directly comparing to used_engine.